modules/codemod/lib/v8/spec/removeDefaultImports.spec.ts

```diff
@@ -69,15 +69,15 @@ describe('renameIconRefs', () => {
       import Breadcrumbs from '@workday/canvas-kit-preview-react/breadcrumbs';
       import ColorPicker from '@workday/canvas-kit-preview-react/color-picker';
       import Menu from '@workday/canvas-kit-preview-react/menu';
-      import Select from '@workday/canvas-kit-preview-react/select';
```

Comment:
yeah any codemod files in previous versions shouldn't be touched because the component still exists in these versions.

Reply:
Ohh gotcha.